File: ts/iq/waffle~get~only_if_suspended+timestamp.ts
/** <iq type="get" xmlns="waffle">. Replies with one of: Success, Error. */
export function buildWaffleForceDeleteState(smaxId: number): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'waffle', smax_id: smaxId, to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'timestamp', attrs: {  } },
      { tag: 'only_if_suspended', attrs: {  } }, /* optional */
    ] }
}

File: rust/iq/waffle~get~only_if_suspended+timestamp.rs
/// <iq type="get" xmlns="waffle">. Replies with one of: Success, Error.
pub fn build_waffle_force_delete_state(smaxId: i64) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "waffle")
        .attr("smax_id", smaxId)
        .attr("to", "s.whatsapp.net")
        .attr("type", "get")
        .child(NodeBuilder::new("timestamp").build())
        .child(NodeBuilder::new("only_if_suspended").build()) // optional
        .build()
}